analytics.js

```diff
@@ -314,11 +322,11 @@ class Analytics {
         (intg) => !intg.isReady || intg.isReady()
       )
     ) {
-        // Integrations are ready
-        // set clientIntegrationsReady to be true
-        object.clientIntegrationsReady = true;
-        // Execute the callbacks if any
-        object.executeReadyCallback();
+      // Integrations are ready
+      // set clientIntegrationsReady to be true
+      object.clientIntegrationsReady = true;
```

Comment:
Where is this defined? Seems like the changes from v1 have not been merged properly.

Reply:
Why is it not showing in the diff then?

Reply:
nm...the diff is broken.